packages/common/src/core/schemas/internal/metrics/MetricSchema.ts

```diff
@@ -20,7 +20,12 @@ export const MetricSchema: IConfigurationSchema = {
       type: "string",
     },
     value: {
+      type: ["number", "string"],
```

Comment:
I think we could consider using conditional schemas (e.g. if/else/then) based on the value of `valueType` to make this `type` more explicit: 

```ts
{
  type: "object",
  properties: {
    ...,
    value: {},
    valueType: {
      type: "string",
      default: "string",
      enum: ["string", "number", "date"],
    },
    ...
  },
  if: {
    properties: { valueType: { enum: ["string", "date"] } },
  },
  then: {
    properties: { value: { type: "string" } },
  },
  else: {
    properties: { value: { type: "number" } },
  }
}
```